pulsar-package-management/core/src/main/java/org/apache/pulsar/packages/management/core/impl/PackagesManagementImpl.java

```diff
@@ -248,7 +248,7 @@ private String metadataPath(PackageName packageName) {
     }
 
     private String packagePath(PackageName packageName) {
-        return packageName.toRestPath();
+        return packageName.toRestPath() + "/pack";
```

Comment:
BTW, could you please add a test to cover this case?

Reply:
Done, we should delete the `packageName.toRestPath()` path at the same time otherwise `list-versions` command will print this parent path (i.e. version path like `v1.0`).